src/hdnode.js

```diff
@@ -72,7 +74,7 @@ HDNode.fromBase58 = function (string, networks) {
   }
 
   if (version !== network.bip32.private &&
-    version !== network.bip32.public) throw new Error('Invalid network')
+    version !== network.bip32.public) throw new Error('Invalid network version')
```

Comment:
Eh,  alignment isn't that big a deal IMHO.
Its also very hard to enforce semantically so I don't bother.